src/StreamJsonRpc.Tests/TargetObjectEventsTests.cs

```diff
@@ -120,6 +120,26 @@ public void EventsAreNotOfferedAsTargetMethods()
         serverRpc.AddLocalRpcTarget(this.server, new JsonRpcTargetOptions { MethodNameTransform = methodNameTransform });
     }
 
+    [Fact]
+    public void NameTransformIsUsedWhenRaisingEvent()
+    {
+        bool eventNameTransformSeen = false;
+        Func<string, string> methodNameTransform = name =>
+        {
+            if (name == nameof(Server.ServerEvent))
+            {
+                eventNameTransformSeen = true;
+            }
+
+            return name;
+        };
+
+        var serverRpc = new JsonRpc(this.serverStream, this.serverStream);
+        serverRpc.AddLocalRpcTarget(this.server, new JsonRpcTargetOptions { MethodNameTransform = methodNameTransform });
+        this.server.TriggerEvent(EventArgs.Empty);
+        Assert.True(eventNameTransformSeen);
```

Comment:
This doesn't actually assert that the notify message used the result of the method transform. And if it did, your transform doesn't actually change the string. Can you revise this test so that it actually verifies that?